### DISCUSSION TITLE
Updating pg_cron to above 1.5

### DISCUSSION THREAD

**Discussion by J4M35K** [supabase/supabase#21962] [Questions] [answered]
Apparently in the new pg_cron update in February of 2023 they introduced sub minute scheduling (1-59 seconds) which would be very useful for many projects utilizing supabase

See here:
https://github.com/citusdata/pg_cron/pull/235

**Comment by J4M35K** [comment]
When will I be able to use this merged 1.6.2 version of the extension
https://github.com/supabase/postgres/pull/900
GitHub
feat: pg_cron 1.6.2 by soedirgo · Pull Request #900 · supabase/post...
What kind of change does this PR introduce?

upgrade pg_cron from v1.4.2 to v1.6.2

Additional context
An upgrade to v1.5 was reverted because of a breaking change, but the breaking change is handl...

**Reply by GaryAustin1** [reply, marked as answer]
It looks available now on hosted platforms.
![image](https://github.com/supabase/supabase/assets/54564956/d15da148-093e-4eb2-8bae-b4ebc7f897f5)
Look under the infrastructure tab.   Extensions are pulled in with upgrades.
